web/src/components/molecules/Common/Form/GroupItem/index.tsx

```diff
@@ -121,24 +121,36 @@ const GroupItem: React.FC<Props> = ({
   const itemGroupId = useMemo(() => value ?? "", [value]);
 
   return (
-    <Collapse collapsible="header" defaultActiveKey={["1"]} style={{ width: 500 }}>
+    <StyledCollapse defaultActiveKey={["1"]}>
       <Panel
         header={parentField?.title + (order !== undefined ? ` (${order + 1})` : "")}
         key="1"
         extra={
           order !== undefined && (
             <>
-              <Icon
-                icon="arrowUp"
-                style={{ marginRight: 10, display: disableMoveUp ? "none" : "inline-block" }}
-                onClick={onMoveUp}
-              />
-              <Icon
-                icon="arrowDown"
-                style={{ marginRight: 10, display: disableMoveDown ? "none" : "inline-block" }}
-                onClick={onMoveDown}
-              />
-              <Icon icon="delete" style={{ marginRight: 10 }} onClick={onDelete} />
+              <IconWrapper
+                disabled={disableMoveUp}
+                onClick={e => {
+                  onMoveUp?.();
+                  e.stopPropagation();
+                }}>
+                <Icon icon="arrowUp" />
+              </IconWrapper>
+              <IconWrapper
+                disabled={disableMoveDown}
+                onClick={e => {
+                  onMoveDown?.();
+                  e.stopPropagation();
+                }}>
+                <Icon icon="arrowDown" />
+              </IconWrapper>
+              <IconWrapper
+                onClick={e => {
+                  onDelete?.();
+                  e.stopPropagation();
+                }}>
+                <Icon icon="delete" />
+              </IconWrapper>
```

Comment:
Just a suggestion, it would be better to move the onClick functions from here and create functions outside the return block.
```suggestion
              <IconWrapper
                disabled={disableMoveUp}
                onClick={handleMoveUp}}>
                <Icon icon="arrowUp" />
              </IconWrapper>
              <IconWrapper
                disabled={disableMoveDown}
                onClick={handleMoveDown}>
                <Icon icon="arrowDown" />
              </IconWrapper>
              <IconWrapper
                onClick={handleDelete}>
                <Icon icon="delete" />
              </IconWrapper>
```

Reply:
Thank you, I think so, too. I'll fix it.